Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,80 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            //Set variable
+            bool backtotop = true;
+            
+
+            //Start While loop
+            while (backtotop)
+
+            {
+                string option;
+                backtotop = true;
+
+                //USer data input
+
+                Console.WriteLine("What is your first name?");
+                var firstname = Console.ReadLine();
+
+                Console.WriteLine("What is your second name?");
+                var secondname = Console.ReadLine();
+
```

Comment:
Don't mix different styles. It will confuse you and other programmers. If you decide to use var, do it for all the base types or use the explicit types else.

Reply:
Using var shouldn't be determined based on "do it everywhere or nowhere". A better (but not the only) way to determine whether or not to use var, is to base that on whether or not the type of the assigned value is explicit. An example:
`var x = 25;` 
This is explicit because 25 is an int. However, when calling methods you are not always aware of its type. That's where you can choose to use a base type.

Besides that, some companies force you to use base types everywhere. I've never heard of a convention to _always_ use var, simply because it hurts understandability.

Reply:
Yeah I was a bit unclear here. What I meant was that if you started using var you should continue doing so as long as there is no good reason to use the explicit one. Like using the explicit one should highlight something rather than being a random choice.